plugins/search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-search",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
Seen as we use `changesets` for releases it mean you don't have to worry about updating these version numbers, so you can also rollback this to `HEAD` too 👍